app/src/main/java/tech/ula/ui/FilesystemEditFragment.kt

```diff
@@ -83,30 +81,98 @@ class FilesystemEditFragment : Fragment() {
         }
     }
 
+    fun setupTextInputs() {
+        input_filesystem_name.setText(filesystem.name)
```

Comment:
Does `setText` need to be invoked here, or is it possible to Kotlin property access syntax?

Reply:
I tried to do `input_filesystem_name.text = filesystem.name` here but it seems that it expects an `Editable` rather than a string.

There are users that recommend using `setText()` to change the text.
https://stackoverflow.com/questions/44493908/setting-text-in-edittext-kotlin/44493995#44493995

Reply:
Yeah, you're totally right. `TextViews` can be set the easier way but not editable ones.